res/css/views/elements/_Tooltip.scss

```diff
@@ -60,7 +60,8 @@ limitations under the License.
     font-weight: 500;
```

Comment:
I can't remove the `element-web` reviewer until a PR is ready nowadays?

Seems to be locked 🤔

![](https://user-images.githubusercontent.com/558581/149611525-412b7a82-b388-4f89-a0fc-d5f4abb676ae.png)

Reply:
not sure why - was this opened as a draft from the start or converted to a draft?

Reply:
I think it was created normally. And I see the system note for converting to draft here:

![](https://user-images.githubusercontent.com/558581/149613496-72bcbc18-fc29-4e36-8dc6-67405c74b9a4.png)

Reply:
Maybe it's something to do with the code owner but I don't think I did anything do that:

![](https://user-images.githubusercontent.com/558581/149613705-d937f661-d5dd-4967-bab7-fe7e9e26aeb0.png)